### DISCUSSION TITLE
Cannot clear filter when using isIndividual: true

### DISCUSSION THREAD

**Discussion by maxacarvalho** [filamentphp/filament#5835] [Help] [answered]
Hi,

I'm, using the table filters with the `isIndividual: true` option but, when searching for a term that returns no results the whole table goes empty, even the columns are gone so the only way to bring the data back is to refresh the page.
I wonder if that's a bug or if it's intended to be like that?

<img width="1274" alt="image" src="https://user-images.githubusercontent.com/45166618/222585911-0dbae703-f7da-4a40-a9df-6ff9a74e09ff.png">

<img width="1253" alt="image" src="https://user-images.githubusercontent.com/45166618/222585959-256d959a-7bb8-4c55-8d09-f0416b1dfae0.png">

**Comment by joaovdiasb** [comment, marked as answer]
Update your filament, now you can clear
![CleanShot 2023-03-02 at 21 02 53@2x](https://user-images.githubusercontent.com/51106311/222594599-eae66570-3e34-4c3f-9ea4-3467209b77e2.png)
